### ISSUE TITLE
Broken links

### ISSUE THREAD

**Issue by OrangeMan-432** [FlameDevV2/FlameDevV2.github.io#1]
https://flamedevv2.github.io/index.html and https://flamedevv2.github.io/contact.html are linked, but go to a 404 error page. This is due to the index.html file being named Index.html, and contact.html not existing.

**Comment by catfoolyou** [comment]
He's a skid. He doesn't even know how Github pages works.

**Comment by OrangeMan-432** [comment]
> He's a skid. He doesn't even know how Github pages works.

Yeah, I know that. I'm just trying to help him out fixing something instead of just calling him a skid like everyone else, then something actually productive may come out of it instead of just wasted time.
